client/src/locales/de.ts

```diff
@@ -281,6 +281,14 @@ export default {
 			"Administratoren haben alle Rechte. Der Raumbesitzer ist automatisch Administrator und kann nicht degradiert werden.",
 		"viewing-as": "Ansicht als",
 		"permission": "Berechtigung",
+		"roles": {
```

Comment:
Don't put English in other languages. It automatically falls back to English if it's not present.